src/mpid/ch4/netmod/ofi/ofi_progress.h

```diff
@@ -73,7 +73,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_NM_progress(int vci, int blocking)
 {
     int mpi_errno = MPI_SUCCESS;
     struct fi_cq_tagged_entry wc[MPIDI_OFI_NUM_CQ_ENTRIES];
-    ssize_t ret;
+    ssize_t num;
```

Comment:
This is very minor -- any idea why `ssize_t` instead of `int`?

Reply:
That's the return type of `fi_cq_read`. perhaps we should update `MPIDI_OFI_get_buffered` to use the same. Or we could use a different variable scoped to block where each is called.

Reply:
I see. Personally, I would add a cast to the return of `fi_cq_read()` than to let odd decisions/conventions from other packages spread into our chaos.

Reply:
Lots of libfabric functions return ssize_t. I believe this concept was adopted from C read/write functions. Negative values are errors, positive is some meaningful value that is potentially large.